datasets/Met-Office-Regional-average-climate-observations-UK-annual-rainfall-2021/info.json

```diff
@@ -1,22 +1,44 @@
 {
+    "$schema": "http://gss-cogs.github.io/family-schemas/dataset-schema-1.1.0.json",
     "id": "met-office-regional-average-climate-observations-uk-annual-rainfall-2021",
     "title": "Regional average climate observations UK annual rainfall 2021",
     "publisher": "Meteorological Office",
-    "description": "",
-    "landingPage": "raw.csv",
+    "description": "Priority dataset for Climate Change Platform project.",
+    "landingPage": "Not available",
     "datasetNotes": [
         "No landing page for this dataset as it is a bespoke dataset supplied to IDP-D as part of the Climate Change v2.0 project."
     ],
-    "published": "",
+    "published": "2022-11-01",
     "families": [
-        "Climate Change"
+        "Climate-Change"
     ],
     "extract": {
-        "source": ""
+        "source": "CSV"
     },
     "transform": {
         "airtable": "recLqYLz2Rx1krPF1",
-        "main_issue": 548
+        "main_issue": 548,
+        "columns": {
+            "Year": {
+                "type": "dimension",
```

Comment:
Change the definition of the year dimension to the following : 
```
"Year": {
    "type": "dimension",
    "new": {
        "subPropertyOf": "http://purl.org/linked-data/sdmx/2009/dimension#refPeriod",
        "codelist": true
    },
    "value": "http://reference.data.gov.uk/id/year/{+year}"
},
```

Since it seems to be a time series we want to make sure the labels appear and the user can filter by each date.